radicle-httpd/src/api/v1/sessions.rs

```diff
@@ -103,6 +106,22 @@ async fn session_signin_handler(
     Err(Error::Auth("Session already authorized"))
 }
 
+/// Delete session.
+/// `DELETE /sessions/:id`
+async fn session_delete_handler(
+    State(ctx): State<Context>,
+    AuthBearer(token): AuthBearer,
+    Path(session_id): Path<String>,
```

Comment:
I kept the `session_id` in the route path, since we are deleting a particular session id